packages/storybook/package.json

```diff
@@ -58,7 +58,8 @@
     "storybook": "start-storybook -p 9009 -s public",
     "build-storybook": "NODE_OPTIONS='--max-old-space-size=4096' build-storybook -s public --quiet",
     "build-storybook:watch": "NODE_OPTIONS='--max-old-space-size=4096' build-storybook -w -s public --quiet",
-    "test": "NODE_ENV=test jest"
+    "test": "NODE_ENV=test jest",
+    "cypress:open": "cypress open"
```

Comment:
@sanketshevkar The script name should rather reflect the work associated with the script so would be better to change the name to `test:cypress` or `test:e2e`.